cli/src/commands/auth/login.ts

```diff
@@ -1,37 +1,50 @@
+import { Flags } from '@oclif/core';
+import { parseProviderString } from '@xata.io/client';
 import { BaseCommand } from '../../base.js';
-import { setProfile } from '../../credentials.js';
+import { hasProfile, setProfile } from '../../credentials.js';
 
 export default class Login extends BaseCommand {
   static description = 'Authenticate with Xata';
 
   static examples = [];
 
   static flags = {
-    ...BaseCommand.forceFlag('Overwrite existing credentials if they exist')
+    ...BaseCommand.forceFlag('Overwrite existing credentials if they exist'),
+    ...BaseCommand.profileFlag,
+    host: Flags.string({
+      description: 'Xata API host provider'
+    })
   };
 
   static args = [];
 
   async run(): Promise<void> {
     const { flags } = await this.parse(Login);
-    const existingProfile = await this.getProfile(true);
-    if (existingProfile) {
+
+    const profile = await this.getProfile(true);
+    const loggedIn = await hasProfile(profile.name);
+    if (loggedIn) {
       const { overwrite } = await this.prompt(
         {
           type: 'confirm',
           name: 'overwrite',
-          message: 'Authentication is already configured, do you want to overwrite it?'
+          message: `Authentication is already configured for ${profile.name} profile, do you want to overwrite it?`
         },
         flags.force
       );
       if (!overwrite) this.exit(2);
     }
 
+    const host = parseProviderString(flags.host);
+    if (!host) {
+      this.error('Invalid host provider, expected either "production", "staging" or "apiUrl,workspacesUrl"');
```

Comment:
```suggestion
      this.error('Invalid host provider, expected either "production", "staging" or "{apiUrl},{workspacesUrl}"');
```

To make it clear that the example are placeholders.